Tests/test_strformat.py

```diff
@@ -629,7 +629,7 @@ def test_int___format__(self):
                 (-2,   'x= 10',   '-xxxxxxxx2'),
                 (-2,   'x^ 10',    'xxxx-2xxxx'),
                 (-2,   'x^ 9',     'xxx-2xxxx'),
-                (2,    '\0^ 9',    '    2    '),
+                (2,    '\0^ 9',    '    2    ' if is_cli else '\0\0\0 2\0\0\0\0'),
```

Comment:
Is there an Issue for this?

Reply:
Nope, but it looks like the fix is simple so I'll include it in this PR.